src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -110,16 +109,74 @@ function dispatchLayersFromUrlIntoStore(store, urlParamValue) {
                 )
             }
             log.debug(`  Add layer ${parsedLayer.id} to active layers`, layerObject)
+            if (layerObject.customAttributes?.features) {
+                layerObject.customAttributes.features
+                    .toString()
+                    .split(':')
+                    .forEach((featureId) => {
+                        featuresRequests.push(
+                            getFeature(
+                                store.getters.getLayerConfigById(parsedLayer.id),
+                                featureId,
+                                store.state.position.projection,
+                                store.state.i18n.lang
+                            )
+                        )
+                    })
+            }
         }
         return layerObject
     })
+
     promisesForAllDispatch.push(
         store.dispatch('setLayers', { layers: layers, dispatcher: STORE_DISPATCHER_ROUTER_PLUGIN })
     )
+    if (featuresRequests.length > 0) {
+        getAndDispatchFeatures(featuresRequests, store) // all features after promise
+    }
 
     return Promise.all(promisesForAllDispatch)
 }
 
+async function getAndDispatchFeatures(featuresPromise, store) {
+    const features = []
+    try {
+        const featuresReceived = await Promise.all(featuresPromise)
+        if (featuresReceived.length > 0) {
+            featuresReceived.forEach((feature) => {
+                features.push(feature)
+            })
+
+            store.dispatch('setSelectedFeatures', {
```

Comment:
so either return the Promise (among others) of this dispatch, or add `await` so that the global Promise of this async func wait on the result